packaging/selinux/flightctl_agent.te

```diff
@@ -251,3 +254,11 @@ allow flightctl_agent_t sysctl_irq_t:dir { write mounton };
 dev_rw_generic_chr_files(flightctl_agent_t)
 dev_rw_generic_blk_files(flightctl_agent_t)
 allow flightctl_agent_t device_t:dir { mounton read getattr search };  
+
+# This allows installation of dnf packages
```

Comment:
even though this is a ro filesystem using installing `--transient` packages is a very real use-case for debug.